docs/docs/recipes/gitlab-continuous-integration.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Recipes: Continuous Integration on Gitlab"
+tableOfContentsDepth: 1
+---
+
+Push your code to gitlab and automate your production build!
+
+### Prerequisites
+
+- A [Gatsby site](/docs/quick-start)
+- A [Gitlab](https://gitlab.com/) account 
+
+### Directions
+
+1. Stop your development server if it is running (`Ctrl + C` on your command line in most cases)
```

Comment:
Can we add this as a note after setting up a Gatsby site steps? see https://www.gatsbyjs.org/docs/recipes/working-with-themes#directions for reference.

Reply:
Sure. Will update in a bit!

Reply:
Included these steps in this commit - https://github.com/gatsbyjs/gatsby/pull/23367/commits/6b92831f7ca91b44be1c1db76cac482dadf40683